src/app/courses-page/course/course.component.scss

```diff
@@ -115,3 +115,20 @@
   display: flex;
   align-items: center;
 }
+
+::ng-deep {
```

Comment:
please avoid ng-deep. Move styles to global file named `_.overlay.scss`.

Reply:
Moved.
Thanks)